src/Files.App/UserControls/NavigationToolbar.xaml

```diff
@@ -342,6 +333,111 @@
+						ItemsSource="{x:Bind ViewModel.PathComponents, Mode=OneWay}">
+						<controls:BreadcrumbBar.RootItem>
+							<Image
+								Width="16"
+								Height="16"
+								Source="/Assets/FluentIcons/Home.png" />
+						</controls:BreadcrumbBar.RootItem>
+						<controls:BreadcrumbBar.ItemTemplate>
+							<DataTemplate x:DataType="dataitems:PathBoxItem">
+								<controls:BreadcrumbBarItem Content="{x:Bind Title, Mode=OneWay}" />
+							</DataTemplate>
+						</controls:BreadcrumbBar.ItemTemplate>
+					</controls:BreadcrumbBar>
+				</controls:OmnibarMode.ContentOnInactive>
+				<controls:OmnibarMode.SuggestionItemTemplate>
+					<DataTemplate x:DataType="datamodels:OmnibarPathModeSuggestionModel">
+						<TextBlock Text="{x:Bind DisplayName, Mode=OneWay}" />
+					</DataTemplate>
+				</controls:OmnibarMode.SuggestionItemTemplate>
+			</controls:OmnibarMode>
+
+			<controls:OmnibarMode
+				IconOnActive="{controls:ThemedIconMarkup Style={StaticResource App.ThemedIcons.Omnibar.Commands}, IsFilled=True}"
+				IconOnInactive="{controls:ThemedIconMarkup Style={StaticResource App.ThemedIcons.Omnibar.Commands}, IconType=Outline}"
+				ModeName="Palette"
+				PlaceholderText="Enter a palette command...">
+				<!--<controls:OmnibarMode.SuggestionItemTemplate>
+					<DataTemplate x:DataType="data:OmnibarPaletteSuggestionItem">
+						<Grid Height="48" ColumnSpacing="12">
+							<Grid.ColumnDefinitions>
+								<ColumnDefinition Width="Auto" />
+								<ColumnDefinition Width="*" />
+								<ColumnDefinition Width="Auto" />
+							</Grid.ColumnDefinitions>
+							<controls:ThemedIcon
+								Width="20"
+								Height="20"
+								VerticalAlignment="Center"
+								Style="{StaticResource App.ThemedIcons.Actions.Copying}" />
+							<StackPanel Grid.Column="1" VerticalAlignment="Center">
+								<TextBlock
+									Style="{StaticResource BodyStrongTextBlockStyle}"
+									Text="{x:Bind Title}"
+									TextTrimming="CharacterEllipsis"
+									TextWrapping="NoWrap" />
+								<TextBlock
+									Foreground="{ThemeResource TextFillColorSecondaryBrush}"
+									Style="{StaticResource CaptionTextBlockStyle}"
+									Text="{x:Bind Description}"
+									TextTrimming="CharacterEllipsis"
+									TextWrapping="NoWrap" />
+							</StackPanel>
+							<StackPanel Grid.Column="2" VerticalAlignment="Center">
+								<TextBlock
+									Text="{x:Bind HotKeys}"
+									TextTrimming="CharacterEllipsis"
+									TextWrapping="NoWrap" />
+							</StackPanel>
+						</Grid>
+					</DataTemplate>
+				</controls:OmnibarMode.SuggestionItemTemplate>-->
+			</controls:OmnibarMode>
+
+			<controls:OmnibarMode
+				IconOnActive="{controls:ThemedIconMarkup Style={StaticResource App.ThemedIcons.Omnibar.Search}, IsFilled=True}"
+				IconOnInactive="{controls:ThemedIconMarkup Style={StaticResource App.ThemedIcons.Omnibar.Search}, IconType=Outline}"
+				ModeName="Search"
+				PlaceholderText="Enter a search query..." />
```

Comment:
Is this supposed to use a localized string?

Reply:
Yeah Ill make them string resources right before getting this ready.

Reply:
Thank you